src/app/src/mainwindow.h

```diff
@@ -305,7 +305,7 @@ class MainWindow : public QMainWindow {
                 m_curState = (stopAcquisition_LiveViewRunning()) ? LiveViewRunning : Error;
             }},
             { {LiveViewAcquisitionRunning, PostProcessing}, [this]() {
-                m_curState = (startPostProcessing_LiveViewRunning()) ? PostProcessingLiveView : Error;
+                m_curState = (startPostProcessing_LiveViewRunning()) ? PostProcessing : Error;
```

Comment:
if this remains `PostProcessingLiveView `, then `postProcessingDone_LiveViewRunning` gets called. This change calls `postProcessingDone`

Reply:
should I remove `postProcessingDone_LiveViewRunning` and `PostProcessingLiveView`? or leave them for now?

Reply:
I'd leave it for now in case they change their minds about it